docs/options.md

```diff
@@ -77,7 +77,7 @@ $ klee --search=random-path demo.o
 
 runs it using the random path selection strategy. The full list of options is shown in KLEE's help message:
 
-{% highlight bash %}
+```
```

Comment:
\`\`\`bash

Reply:
That was on purpose. The parser for highlighting is broken and the result looks awful.